python-base/ubuntu20.04-python3.9/docker/scripts/install_python.sh

```diff
@@ -5,11 +5,16 @@ set -exuo pipefail
 
 # Install python
 apt-get install -y --no-install-recommends \
-  "python${APT_PYTHON_VERSION}" \
+  "python${PYTHON_VERSION}" \
   python3-distutils
 
 # Set python active
-update-alternatives --install /usr/bin/python python /usr/bin/python3.9 10
+update-alternatives --install /usr/bin/python python "/usr/bin/python${PYTHON_VERSION}" 10
+update-alternatives --install /usr/bin/python3 python3 "/usr/bin/python${PYTHON_VERSION}" 10
```

Comment:
Quite a lot of things, including Poetry, actually default to `/usr/bin/python3` so this was pretty important